tests/PHPStan/Rules/Exceptions/data/bug-9066.php

```diff
@@ -8,6 +8,7 @@ public function mayThrow()
 	{	$map = new \Ds\Map();
 		try {
 			$map->get('1');
+			$map->remove('1');
```

Comment:
Please add a separate method for testing `remove` where it's supposed to throw an exception.